observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc

```diff
@@ -0,0 +1,146 @@
+
+== Attribute mapping
+
+When the Loki Operator is set to `openshift-logging` mode, it automatically applies a default set of xref:../../../observability/logging/logging-6.0/log6x-opentelemetry-data-model.adoc#attributes[attribute mappings]. These mappings align specific OTLP attributes with Loki's stream labels and structured metadata.
+
+For typical setups, these default mappings should be sufficient. However, you might need to customize attribute mapping in the following cases:
+
+* Using a custom Collector: If your setup includes a custom collector that generates additional attributes, consider customizing the mapping to ensure these attributes are retained in Loki.
+* Adjusting attribute detail levels: If the default attribute set is more detailed than necessary, you can reduce it to essential attributes only. This can avoid excessive data storage and streamline the logging process.
+
+For more infomation on configuring custom attribute mappings, see xref:../../../observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc#custom-attribute-mapping-for-openshift[Custom attribute mapping for OpenShift].
+
+[NOTE]
+====
+Attributes that are not mapped to either stream labels or structured metadata are not stored in Loki.
+====
+
+=== Custom attribute mapping for OpenShift
+
+When using the Loki Operator in `openshift-logging` mode, attribute mapping follow OpenShift defaults, but custom mappings can be configured to adjust these. Custom mappings allow further configurations to meet specific needs. 
+
+In `openshift-logging` mode, custom attribute mappings can be configured globally for all tenants or for individual tenants as needed. When custom mappings are defined, they are appended to the OpenShift defaults. If default recommended labels are not required, they can be disabled in the tenant configuration. For more information, see xref:../../../observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc#customizing-openshift-defaults[Customizing OpenShift defaults].
+
+[NOTE]
+====
+A major difference between the Loki Operator and Loki itself lies in inheritance handling. Loki only copies `default_resource_attributes_as_index_labels` to tenants by default, while the Loki Operator applies the entire global configuration to each tenant in `openshift-logging` mode.
+====
+
+Within `LokiStack`, attribute mapping configuration is managed through the `limits` setting:
+
+[source,yaml]
+----
+# [...]
+spec:
+  limits:
+    global:
+      otlp: {} # <1> 
+    tenants:
+      application:
+        otlp: {} # <2> 
+----
+<1> Global OTLP attribute configuration.
+<2> OTLP attribute configuration for the `application` tenant within `openshift-logging` mode.
+
+[NOTE]
+====
+Both global and per-tenant OTLP configurations can map attributes to stream labels or structured metadata. At least _one stream-label_ is required to save a log entry to Loki storage, so ensure this configuration meets that requirement.
```

Comment:
Is the emphasis really needed here? Also this is the first instance I see of "stream label" having a `-` in the middle:
```suggestion
Both global and per-tenant OTLP configurations can map attributes to stream labels or structured metadata. At least one stream label is required to save a log entry to Loki storage, so ensure this configuration meets that requirement.
```